platform/30-kyverno-setup.sh

```diff
@@ -13,14 +13,14 @@ KYVERNO_RESOURCE_DIR=${GIT_ROOT}/resources/kyverno/admission-control-policy
 # Define variables.
 C_GREEN='\033[32m'
 C_YELLOW='\033[33m'
-C_RED='\033[31m'
 C_RESET_ALL='\033[0m'
 
 # Wait until pods are ready.
 # $1: namespace, $2: app label
 wait_for_pods () {
+  sleep 1
```

Comment:
Small wait to prevent a race condition where the following query doesn't find any pods even though one has just been deployed.

Reply:
So it's a wait, to make sure the wait can see the pods?

Reply:
yep.